### ISSUE TITLE
v6 high order components problem

### ISSUE THREAD

**Issue by gabrielenosso** [erikras/redux-form#874]
Hi, with the v6 version, I have problems using a redux-form with the redux "connect".

Practically if I use 2 high order components (one for the redux store using "connect" and one for redux-form using "reduxForm"), it doesn't work if I use redux-for as the first one.
But it works if I use it as the second one (It was working in previous versions).

The following code doesn't work (the component is not updated when the state.data object is updated in the redux store)

```js
const MyComponentForm = reduxForm({
  form: 'eventsFilter'   // Unique name for this form (to register it in the Redux global store)
})(MyComponent);

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(MyComponentForm);
```

While the following code works:

```js
const MyComponentConnect = connect(
  mapStateToProps,
  mapDispatchToProps
)(MyComponent);

export default reduxForm({
  form: 'eventsFilter'   // Unique name for this form (to register it in the Redux global store)
})(MyComponentConnect);
```

**Comment by erikras** [comment]
Hmmm.. it's working in [the example](http://redux-form.com/6.0.0-alpha.6/examples/initializeFromState/).

I'd need to see the code of your `mapStateToProps` and `mapDispatchToProps` functions to help further.

**Comment by gabrielecirulli** [comment]
While browsing I found this might be related to the issue I just reported as well: https://github.com/erikras/redux-form/issues/906

**Comment by nktssh** [comment]
seems to be fixes in `v6.0.0-alpha-7`, as commented in #906

**Comment by erikras** [comment]
Closing based on @nktssh's comment.
